### ISSUE TITLE
godot error cant open scene

### ISSUE THREAD

**Issue by VagAttack2** [godotengine/godot#88732]
### Tested versions

version 3.5.1 stable

### System information

godot 3.5.1 stable

### Issue description

So im a new dev and i started developing a game at the godot game engine now day out of nowhere i opend my project and i could every script .gd and every tscn scene except form one my main one called level_1.tcsn. For one reason its blue and when i click it it says this message
scene/resources/resource_format_text.cpp:804 - res://level_1.tscn:1 - Parse Error:
Failed loading resource: res://level_1.tscn. Make sure resources have been imported by opening the project in the editor at least once.
editor/editor_data.cpp:531 - Index p_idx = 1 is out of bounds (edited_scene.size() = 1).

PLEASE I NEED HELP. i cant open the file
im at 70% on my project to complete and release it so it would be great if someone could help me
With a lot of appreciation vag attack.

### Steps to reproduce

i did use a notepad++ but the only thing i found was 1 line with many words NULL NULL NULL like that

### Minimal reproduction project (MRP)

![icon](https://github.com/godotengine/godot/assets/160927928/683a72e9-5104-4b04-8c8d-972e646527d3)

**Comment by AThousandShips** [comment]
Please upload a project or some further information, this is not enough to test this problem

**Comment by VagAttack2** [comment]
> Please upload a project or some further information, this is not enough to test this problem

yeah like i said im a new dev evrything here is so confusing can you plesase guide me i tried to send the scene but i was having a problem because its tscn file

**Comment by AThousandShips** [comment]
Do like this:
- A small Godot project which reproduces the issue, with no unnecessary files included. Be sure to not include the `.godot` folder in the archive (but keep `project.godot`).
- Having an MRP is very important for contributors to be able to reproduce the bug in the same way that you are experiencing it. When testing a potential fix for the issue, contributors will use the MRP to validate that the fix is working as intended.
- Drag and drop a ZIP archive to upload it (max 10 MB). **Do not select another field until the project is done uploading.**

**Comment by VagAttack2** [comment]
> Do like this:
> 
> * A small Godot project which reproduces the issue, with no unnecessary files included. Be sure to not include the `.godot` folder in the archive (but keep `project.godot`).
> * Having an MRP is very important for contributors to be able to reproduce the bug in the same way that you are experiencing it. When testing a potential fix for the issue, contributors will use the MRP to validate that the fix is working as intended.
> * Drag and drop a ZIP archive to upload it (max 10 MB). **Do not select another field until the project is done uploading.**

[project.zip](https://github.com/godotengine/godot/files/14438856/project.zip)
here it is 
_now im gonna break down my issue and where is it_
**1 locate the file named Level1.tscn in the filesystem**
**2 double tap in the file**
**3 check the output and you will see the error**

Thanks for reading my problem it would be grateful to solve it

**Comment by derksendawa** [comment]
I had this bug once because I had the .tscn file open in the script editor

Make sure you close the .tscn file if you're going to open it in the 2d vis mode

Also if this is not already reported, it would be nice of you to create a further issue about this behaviour.
